django_freeradius/base/admin.py

```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'max_all_time_limit',
+                    'created', 'modified']
+
+    def get_daily_session_limit(self, obj):
+        # returns daily session limit in hours
+        if obj.daily_session_limit:
+            return round(obj.daily_session_limit / 3600, 2)
+    get_daily_session_limit.short_description = 'daily session limit(hours)'
+
+    def get_daily_bandwidth_limit(self, obj):
+        # returns daily bandwidth limit in megabytes
+        if obj.daily_bandwidth_limit:
+            return int(obj.daily_bandwidth_limit / 1000 ** 2)
+    get_daily_bandwidth_limit.short_description = 'daily bandwidth limit(MB)'
```

Comment:
Add a space after limit in short description, i.e., `daily bandwidth limit (MB)`

Reply:
same here